BraveRewardsUI/Tipping/TippingSelectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveRewards
+
+class TippingSelectionViewController: OptionsSelectionViewController {
```

Comment:
A subclass for this one use case is probably over the top, and could have just copied this:

https://github.com/brave/brave-rewards-ios/blob/4fcd743924e9573547aa2bfb3d44cecf5f766d3d/BraveRewardsUI/Settings/Auto-Contribute/Settings/AutoContributeSettingsViewController.swift#L97-L118

If we're going to do this just to get custom appearance for the text, we should just make it generic like `BATSelectionViewController` and use it in auto-contribute settings as well

We'll have to make `OptionsSelectionViewController` deceleration use generics in this case like 

```swift
class OptionsSelectionViewController<OptionType: DisplayableOption>: UIViewController` {
```

Then here you can do:

```swift
class BATValueOptionsSelectionViewController: OptionsSelectionViewController<BATValue> {
```

Then when you make a new designated initializer it won't complain that the types are wrong

If we can't get that part working properly for now or has more issues, lets just scrap it and do the non-styled version from Auto-Contribute Settings for now